README.md

```diff
@@ -54,9 +54,27 @@ Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 --- myhttpsamplehost.com:80 tcp test statistics ---
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
+Time spent in 10 established connections min/avg/max/stdDev = 17.929ms/19.814ms/29.811ms/3.353ms
 % echo $?
 0
 ```
+
+Partial unsuccesful execution
+```bash
+% ./tcpgoon --host myhttpsamplehost.com --port 8080 --connections 10 --sleep 999 -y
+Total: 10, Dialing: 0, Established: 0, Closed: 0, Error: 0, NotInitiated: 10
+Total: 10, Dialing: 9, Established: 1, Closed: 0, Error: 0, NotInitiated: 0
+Total: 10, Dialing: 8, Established: 2, Closed: 0, Error: 0, NotInitiated: 0
+Total: 10, Dialing: 8, Established: 2, Closed: 0, Error: 0, NotInitiated: 0
+Total: 10, Dialing: 8, Established: 2, Closed: 0, Error: 0, NotInitiated: 0
+Total: 10, Dialing: 8, Established: 2, Closed: 0, Error: 0, NotInitiated: 0
+Total: 10, Dialing: 0, Established: 2, Closed: 0, Error: 8, NotInitiated: 0
+--- myhttpsamplehost.com:8080 tcp test statistics ---
+Total: 10, Dialing: 0, Established: 2, Closed: 0, Error: 8, NotInitiated: 0
+Time spent in 2 established connections min/avg/max/stdDev = 212.98ms/720.286ms/1.227592s/507.305ms
+Time spent in 8 failed connections min/avg/max/stdDev = 5.000144s/5.001417s/5.002747s/795µs
```

Comment:
Or maybe just as simple as "Stats for..."

Reply:
I've gone for "Timing stats for..."